shell/circleci/setup.sh

```diff
@@ -44,5 +45,7 @@ config:
 storageURL: git@github.com:getoutreach/box
 EOF
 
+"$LIB_DIR/download-box.sh"
```

Comment:
Yeah, fair call. I always let tools that need it refresh it since the logic is in go... but that was always going to be a little fragile.

We can remove the above stub now that we're doing this

Reply:
we still need the storageURL which the stub above writes out

Reply:
i think its best if the download script doesn't overwrite what is already there - and instead just looks for a storageURL

Reply:
Fair, maybe we should just read it from a CircleCI context instead?

Reply:
i think that just further expands the surface area of how we deal with box. i think its best if we kinda keep it in parity with the go stuff as much as possible - and i think this solution is pretty close